icons/undo.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <polyline transform="translate(.9998 5.9996)" points="1 4 1 10 7 10"/>
+ <path d="M 21.49,15 C 19.2818,8.7598 11.319,6.9582 6.64,11.64 L 2,16"/>
```

Comment:
```suggestion
  <polyline transform="translate(.9998 5.9996)" points="1 4 1 10 7 10"/>
  <path d="M 21.49,15 C 19.2818,8.7598 11.319,6.9582 6.64,11.64 L 2,16"/>
```

Reply:
Fixed the indentation in my third commit.